docs/content/operations/web-consoles.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: doc_page
+title: "Web Consoles"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+# Web Consoles
+
+Druid has a few console UIs that provide cluster management functionality, described below.
+
+## Druid Console
+
+Druid provides a console for managing datasources, segments, tasks, data processes (Historicals and MiddleManagers), and coordinator dynamic configuration. The user can also run SQL and native Druid queries within the console.
+
+To use this console, it is necessary to run a [Router](../development/router.html) process.
+
+In addition, the following cluster settings must be enabled:
+
+- the Router's [management proxy](../development/router.html#enabling-the-management-proxy) must be enabled.
+- the Broker processes in the cluster must have [Druid SQL](../querying/sql.html) enabled.
+
+After enabling Druid SQL on the Brokers and deploying a Router with the managment proxy enabled, the Druid console can be accessed at:
+
+```
+http://<ROUTER_IP>:<ROUTER_PORT>
+```
+
+This console subsumes the functionality provided by the older consoles described below, which are still available if needed.
+
+## Coordinator Consoles
```

Comment:
I think this section shoul dbe called Legacy consoles and emphasize the recommendation to use the new one

Reply:
Also note that we may remove these consoles in the future

Reply:
Added note about possible removal

Reply:
Hm, on second thought, I took out the note on removal, in #6923 I don't think we've quite reached consensus on what to do with the older consoles.

Reply:
IMO, the best way to present the new console is that it is something that we believe is better / easier to use / more capable than the older consoles, so really encourage people to check it out and present it as the 'main' or 'default' option. I'd also be clear that it lives on the Router right now, and in particular that means (a) you need a Router running to use it [link to other docs discussing running Brokers and Routers on a Query server]; and (b) it hasn't replaced the older consoles yet, but it may in the future.

If readers get the above picture then I think the docs are doing their job.

Reply:
Added a link to the `design/processes` page, put in a note about the legacy consoles possibly being removed again